packages/lab/src/stepper-input/StepperInput.tsx

```diff
@@ -32,18 +36,14 @@ export interface StepperInputProps
    * The minimum value that can be selected.
    */
   min?: number;
-  /**
-   * Whether to hide the stepper buttons. Defaults to `false`.
-   */
-  hideButtons?: boolean;
   /**
    * Callback when stepper input loses focus.
    */
   onBlur?: FocusEventHandler<HTMLInputElement>;
   /**
    * Callback when stepper input value is changed.
    */
-  onChange?: (changedValue: number | string) => void;
+  onChange?: (value: number | string, event?: SyntheticEvent) => void;
```

Comment:
any reason why the value is not after the event? i think whenever we have onChanges with an event and a value we tend to have them on the opposite order

Reply:
it's because the event is optional, it is a typescript error the other way round

Reply:
ah i missed the optional, fair enough

Reply:
...just curious, why does it need to be optional?

Reply:
haha this was the whole problem, basically if you click and hold the button there is no event

Reply:
This will result in a different API with others.
Have you tried any different optional approach? Maybe not set the arg being optional, but the actual type being event or undefined? Or make it null?

Reply:
okay yeah I think undefined makes sense because it is actually passed as undefined